src/ui/public/filter_editor/params_editor/filter_params_values_controller.js

```diff
@@ -0,0 +1,32 @@
+
+
+import _ from 'lodash';
+import chrome from 'ui/chrome';
+
+const baseUrl = chrome.addBasePath('/api/kibana/values');
+
+export function filterParamsValuesController($http, $scope, $rootScope) {
```

Comment:
But we are trying to remove reliance on the angular promise service, and the filter editor should be moving to react eventually.  I don't actually think we want to use the angular Promise class here.    https://github.com/elastic/kibana/issues/13855

Reply:
Since this is using a mix of angular promises from `$http` I think it should also return angular promises when it returns early. This is also an angular controller, so I think angular promises are appropriate.